modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -1248,21 +1269,29 @@ class NDB_Form_User_Accounts extends NDB_Form
         //======================================
         //        Validate Examiner Status
         //======================================
-
+        $matched =false;
         foreach ($values as $k=>$v) {
-            if (preg_match("/^ex_[1-9]+$/", $k)
-                && $values['examiner_radiologist'] == ''
-            ) {
-                $errors['examiner_group'] = "Please specify if examiner is a ".
-                    "radiologist";
-            }
-            if (preg_match("/^ex_[1-9]+$/", $k)
-                && $values['examiner_radiologist'] !== ''
-                && $values['examiner_pending'] == ''
-            ) {
-                $errors['examiner_group'] = "Please set pending approval Yes or No";
+            if (preg_match("/^ex_[1-9]+$/", $k)) {
+                $matched =true;
+                if ($values['examiner_radiologist'] == '') {
+                    $errors['examiner_group'] = "Please specify if examiner is a " .
+                        "radiologist";
+
+                }
+                if ($values['examiner_radiologist'] !== ''
+                    && $values['examiner_pending'] == ''
+                ) {
+                    $errors['examiner_group'] = "Please set pending approval Yes ".
+                        "or No";
+                }
             }
         }
+        if (!$matched
```

Comment:
logic added to prevent saving radiologist and pending if no actual examiner site was select 

ie you cant be an radiologist nowhere....